src/parsing.jl

```diff
@@ -154,7 +160,7 @@ _DEFAULT_TYPE_MAP[:numeric] = Decimal
 
 ## character
 # bpchar is char(n)
-function Base.parse(::Type{String}, pqv::PQValue{PQ_SYSTEM_TYPES[:bpchar]})
+function Base.parse(::Type{String}, pqv::PQValue{PQ_SYSTEM_TYPES[:bpchar], false})
```

Comment:
We should have constants instead of Bool literals, e.g.

```julia
const BINARY = true
const TEXT = false
```

or we might consider using one of the many enum approaches.